snippets/reviews.liquid

```diff
@@ -11,7 +11,6 @@
 
 {% javascript %}
   const defaultAvatar = '{{ "avatar.png" | asset_url }}';
-  const productId = '{{ reviews_product_id }}';
 {% endjavascript %}
```

Comment:
don't worry nothing is gonna break if you check product-home.liquid you will find it

Reply:
as i said you you shouldn't delete the product id since this will break the reviews that we have on the home page
instead of removing the id you need just to change the variable name on the product page since we have the same const name on the home page and on the product page